doc/api/path.markdown

```diff
@@ -94,9 +103,11 @@ Windows examples:
 
     path.isAbsolute('//server')  // true
     path.isAbsolute('C:/foo/..') // true
-    path.isAbsolute('bar\\baz')   // false
+    path.isAbsolute('bar\\baz')  // false
     path.isAbsolute('.')         // false
 
+*Note:* If the path is a zero length string then `false` will be returned.
```

Comment:
@brendanashworth Well, in other cases empty string means the current directory, and this is an exception. So, I think it would be better to leave it in.